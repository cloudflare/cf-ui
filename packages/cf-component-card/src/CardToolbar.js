import React from 'react';

import PropTypes from 'prop-types';

class CardToolbar extends React.Component {
  render() {
    return (
      <div className="cf-card__toolbar">
        <div className="cf-card__toolbar_controls">
          {this.props.controls}
        </div>
        <div className="cf-card__toolbar_links" role="tablist">
          {this.props.links}
        </div>
      </div>
    );
  }
}

CardToolbar.propTypes = {
  controls: PropTypes.any,
  links: PropTypes.any
};

export default CardToolbar;
