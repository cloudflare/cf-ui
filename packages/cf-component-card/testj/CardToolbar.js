import React from 'react';
import renderer from 'react-test-renderer';
import CardToolbar from '../src/CardToolbar';

test('should render', () => {
  const component = renderer.create(
    <CardToolbar controls={'Controls'} links={'Links'} />
  );
  expect(component.toJSON()).toMatchSnapshot();
});
