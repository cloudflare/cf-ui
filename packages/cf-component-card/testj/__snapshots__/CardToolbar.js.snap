// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<div
  className="cf-card__toolbar"
>
  <div
    className="cf-card__toolbar_controls"
  >
    Controls
  </div>
  <div
    className="cf-card__toolbar_links"
    role="tablist"
  >
    Links
  </div>
</div>
`;
